src/styles/blocks/day.scss

```diff
@@ -0,0 +1,26 @@
+.calendar__day {
+  width: $day-size;
```

Comment:
Ensure that the `$day-size` variable is defined in your SCSS files. Otherwise, this line will cause a compilation error.